lib/prepareReturnValue.mjs

```diff
@@ -38,32 +31,10 @@ export default function prepareReturnValue({
 		errored,
 		results: [],
 		report: '',
-
-		/**
-		 * @deprecated
-		 * @todo Remove in the next major version.
-		 */
-		get output() {
-			if (!quietDeprecationWarnings && !this._outputWarned) {
-				emitDeprecationWarning(
-					'`output` is deprecated.',
-					'RESULT_OUTPUT_PROPERTY',
```

Comment:
[suggestion] `RESULT_OUTPUT_PROPERTY` will no longer be used.

https://github.com/stylelint/stylelint/blob/b092ef8d5c2c2df08a1e50940cf91c5b62e6e29e/lib/utils/emitWarning.mjs#L7

Not to use it by mistake in the future, what if commenting it out like this?

```diff
-	RESULT_OUTPUT_PROPERTY: '003',
+	// RESULT_OUTPUT_PROPERTY: '003', // Removed in 17.0.0
```